src/Workspaces/Core/Portable/Experiments/IExperimentationService.cs

```diff
@@ -39,7 +39,6 @@ internal static class WellKnownExperimentNames
         public const string OOPServerGC = "Roslyn.OOPServerGC";
         public const string LspTextSyncEnabled = "Roslyn.LspTextSyncEnabled";
         public const string ImportsOnPasteDefaultEnabled = "Roslyn.ImportsOnPasteDefaultEnabled";
-        public const string LspTextSyncEnabled = "Roslyn.LspTextSyncEnabled";
```

Comment:
some bad earlier merge caused this.  it's already defined two lines above, so this was causing a build break